replications/remotewrite/writer.go

```diff
@@ -208,9 +208,14 @@ func PostWrite(ctx context.Context, config *influxdb.ReplicationHTTPConfig, data
 	conf.HTTPClient.Timeout = timeout
 	client := api.NewAPIClient(conf).WriteApi
 
+	bucket := config.RemoteBucketID.String()
+	if config.RemoteBucketName != "" {
+		bucket = config.RemoteBucketName
+	}
+
 	req := client.PostWrite(ctx).
 		Org(config.RemoteOrgID.String()).
-		Bucket(config.RemoteBucketID.String()).
+		Bucket(bucket).
```

Comment:
I hate that there is no distinction in the write API between bucket id and bucket name here.